cli_reference/kn-cli-tools.adoc

```diff
@@ -1,5 +1,4 @@
 :_content-type: ASSEMBLY
-include::_attributes/serverless-document-attributes.adoc[]
```

Comment:
You need to move the `common-attributes.adoc` include here or the attribute in your title won't render. (Ashleigh and I figured this out the hard way earlier.)